docs/mesos/PrepareExternalServices.md

```diff
@@ -0,0 +1,54 @@
+# Prepare External Services for Mesos + Calico Deployment
+*Most Mesos clusters will run these services outside of their core Mesos cluster, separate from the Masters and Agents which read from them.*
```

Comment:
"outside" the cluster sounds a bit strange.  I'd say "Most Mesos clusters will run these services on specific, dedicated machines chosen to maximize availability."